bindings/pyroot/cppyy/CPyCppyy/src/CPPExcInstance.cxx

```diff
@@ -163,7 +234,7 @@ PyTypeObject CPPExcInstance_Type = {
     0,                             // tp_setattr
     0,                             // tp_compare
     (reprfunc)ep_repr,             // tp_repr
-    0,                             // tp_as_number
+    &ep_as_number,                 // tp_as_number
```

Comment:
This was the actual bug?

Reply:
This was unrelated with the corruption. This change (see two commits) it's related with adding boolean semantics to `CPPExcInstance`.

Reply:
I see, thanks!